Service/GithubApiService.php

```diff
@@ -346,65 +346,59 @@ public function getGithubOwnerInfo(array $item): array
     //     return $publiccode;
     // }
 
-    // /**
-    //  * This function is searching for repositories containing a publiccode.yaml file.
-    //  *
-    //  * @param string $slug
-    //  *
-    //  * @throws GuzzleException
-    //  *
-    //  * @return array|null|Response
-    //  */
-    // public function checkPublicRepository(string $slug)
-    // {
-    //     if ($this->checkGithubKey()) {
-    //         return $this->checkGithubKey();
-    //     }
+    /**
+     * This function is searching for repositories containing a publiccode.yaml file.
+     *
+     * @param string $slug
+     *
+     * @return array|null|Response
+     */
+    public function checkPublicRepository(string $slug)
```

Comment:
Dit is niet de laatste versie van deze functie, maar komt goed met mijn pr (neem aan dat deze functie nu nergens anders word gebruikt naast rating)

Reply:
Ja ik had t startje van mij mee gecommit